### ISSUE TITLE
excelHtml5 action save zip files instead of xlsx

### ISSUE THREAD

**Issue by dbrammertz** [DataTables/Buttons#67]
The problem: Saving the excel to disk, Firefox "Save As "dialog indicates that a zip file is going to be saved. This causes an error if  "Open with" is used, since it is not an excel file.

The problematic code is:
```javascript
		_saveAs(
			zip.generate( {type:"blob"} ),
			_filename( config )
		);
```

This code will apply the mime type zip.

Can be easily be patched by 
```javascript
		var dataArray = zip.generate( {type:"uint8array"} );
		_saveAs(
			new Blob([dataArray], { type: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" }),
			_filename( config )
		);
```

As patch
```
@@ -662,8 +662,9 @@
 		xl_rels.file(       'workbook.xml.rels',   excelStrings['xl/_rels/workbook.xml.rels'] );
 		xl_worksheets.file( 'sheet1.xml',          excelStrings['xl/worksheets/sheet1.xml'].replace( '__DATA__', xml ) );
 
+		var dataArray = zip.generate( {type:"uint8array"} );
 		_saveAs(
-			zip.generate( {type:"blob"} ),
+			new Blob([dataArray], { type: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" }),
 			_filename( config )
 		);
 	},
```

**Comment by DataTables** [comment]
This should have been resolved by: 4b4756d927c6daab649b4cc6378c7167d92b4107 and 63d7333034f39f10f6034946b419dc2ae2ebc152.

Can you link to a test case showing the issue that uses the software form this repo (the fixes have not yet been released).

**Comment by dbrammertz** [comment]
Thanks Allan, seems to work: https://jsfiddle.net/dbrammertz/oL0kc8s1/
